scripts/install_odf.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/sh
+
+ibmcloud config --check-version=false
+ibmcloud login -apikey ${IC_API_KEY}
+ibmcloud ks cluster config -c ${CLUSTER} --admin
+
+# Addons variables assigments
+
+
+# Retrieve the openshift cluster version
+ROKS_VERSION=`oc get clusterversion -o jsonpath='{.items[].status.history[].version}{"\n"}'`
+
+# Round the number down to 0 (e.g. 4.7.40 to 4.7.0)
+ROKS_VERSION="${ROKS_VERSION%.*}.0"
+
+ROKS_VERSION_COMP=$(echo "$ROKS_VERSION" | tr -d -c 0-9)
+
+if [ $ROKS_VERSION_COMP -ge 470 ]; then
+    echo "Supported version";
+    ibmcloud oc cluster addon enable openshift-data-foundation -c ${CLUSTER} --version ${ROKS_VERSION} \
```

Comment:
I think this can be changed to `ibmcloud ks cluster addon enable openshift-data-foundation`